services/web/server/tests/unit/with_dbs/02/test_projects_comments_handlers.py

```diff
@@ -0,0 +1,191 @@
+# pylint: disable=protected-access
+# pylint: disable=redefined-outer-name
+# pylint: disable=too-many-arguments
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+
+from typing import Any
+
+import pytest
+from aiohttp import web
+from aiohttp.test_utils import TestClient
+from pytest_simcore.helpers.utils_login import LoggedUser
+from simcore_postgres_database.models.projects import projects
+from simcore_service_webserver._meta import api_version_prefix
+from simcore_service_webserver.db.models import UserRole
+
+API_PREFIX = "/" + api_version_prefix
+
+
+@pytest.mark.parametrize(
+    "user_role,expected",
+    [
+        (UserRole.ANONYMOUS, web.HTTPUnauthorized),
+        (UserRole.GUEST, web.HTTPOk),
+        (UserRole.USER, web.HTTPOk),
+        (UserRole.TESTER, web.HTTPOk),
+    ],
+)
+async def test_project_comments_user_role_access(
+    client: TestClient,
+    logged_user: dict[str, Any],
+    user_project: dict[str, Any],
+    template_project: dict[str, Any],
+    user_role: UserRole,
+    expected: type[web.HTTPException],
+):
+    base_url = client.app.router["list_project_comments"].url_for(
+        project_uuid=user_project["uuid"]
+    )
+    resp = await client.get(base_url)
+    assert resp.status == 401 if user_role == UserRole.ANONYMOUS else 200
+
+
+@pytest.mark.parametrize(
+    "user_role,expected",
+    [
+        (UserRole.USER, web.HTTPOk),
+    ],
+)
+async def test_project_comments_basic_crud_operations(
```

Comment:
MINOR: this is not a basic test (as the name suggest) but rather a full workflow. I would rather rename it and mark it as acceptance test:
```python
@pytest.mark.acceptance_test(
    "https://github.com/ITISFoundation/osparc-issues/issues/993"
)
```

Reply:
@pcrespov how are these acceptance tests handled? how do you run them?